cfme/v2v/migration_plans.py

```diff
@@ -433,16 +433,17 @@ def successful(self):
         view = navigate_to(self, "Complete")
         return view.plans_completed_list.is_plan_succeeded(self.name)
 
-    def get_plan_vm_list(self):
+    def get_plan_vm_list(self, wait_for_migration=True):
         """
         Navigates to plan details and waits for plan to complete
         returns : List of Vm's on plan details page
         """
         view = navigate_to(self, "Details")
         view.wait_displayed()
-        wait_for(func=view.plan_in_progress,
-                 message="migration plan is in progress, be patient please",
-                 delay=5, num_sec=3600)
+        if wait_for_migration:
```

Comment:
Can you just use `self.wait_for_state('In Progress')` here instead?

Reply:
This is from Details Page of migration so the wait for progress methods tracks migration of each VM.
It is different from self.wait_for_state('In Progress')